src/ServiceControl/Operations/AuditImporter.cs

```diff
@@ -18,7 +21,40 @@ public AuditImporterFeature()
 
         protected override void Setup(FeatureConfigurationContext context)
         {
-            context.Container.ConfigureComponent<AuditImporter>(DependencyLifecycle.SingleInstance);
+            var settings = context.Settings.Get<Settings>();
+
+            if(settings.IngestAuditMessages)
+            {
+                context.Container.ConfigureComponent<AuditImporter>(DependencyLifecycle.SingleInstance);
+                context.Container.ConfigureComponent<AuditIngestor>(DependencyLifecycle.SingleInstance);
+
+                context.AddSatelliteReceiver(
+                    "Audit Import",
+                    settings.AuditQueue, 
```

Comment:
Is the `AuditQueue` expected to be logical address or transport address? If logical than we should translate to transport address here.

Reply:
It is meant to be just a logical queue name. What's the best way to do that translation? Like this?

```csharp
var errorQueueLogicalAddress = LogicalAddress.CreateLocalAddress(settings.ErrorQueue, null);
var transportInfrastructure = context.Settings.Get<TransportInfrastructure>();
var errorQueueTransportAddress = transportInfrastructure.ToTransportAddress(errorQueueLogicalAddress);
```

Reply:
Or should I be doing something more like this?

```csharp
var auditInstance = transportInfrastructure.BindToLocalEndpoint(new EndpointInstance(settings.AuditQueue));

var auditLogicalAddress = LogicalAddress.CreateLocalAddress(settings.AuditQueue, auditInstance.Properties);
var auditTransportAddress = transportInfrastructure.ToTransportAddress(auditLogicalAddress);
```

I'm not sure entirely what the transports use the instance properties for